helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -586,6 +586,26 @@ processors:
         new_name: container.memory.usage
   {{- end }}
 
+  {{- if or .Values.autodetect.prometheus .Values.autodetect.istio }}
+  # This processor is used to remove excessive istio attributes to avoid running into the dimensions limit.
+  # This configuration assumes single cluster istio deployment. If you run istio in multi-cluster scenarios,
+  # you may need to adjust this configuration.
+  attributes/istio:
+    include:
+      match_type: regexp
+      metric_names:
+        - istio_.*
+    actions:
```

Comment:
@dmitryax we hit a similar issue when infra upgraded istio in lab recently. This is what I noticed. There were 47 dims on the most common metric `istio_requests_total`. I found the easiest way to fix it was to disable the addition of host/env dimensions on metrics collected from istio-ingress (that freed up like 11 dimensions). In addition to this, infra team also excluded a few tags from being added in envoy itself when generating metrics.

Reply:
We already disabled all the attributes added by the prometheus receiver in https://github.com/signalfx/splunk-otel-collector/pull/3042 and https://github.com/signalfx/splunk-otel-collector-chart/pull/763. So at this point, we are at 36 (and 37 for buckets) dimensions for a typical deployment on GKE. Disabling these attributes gives us enough room I believe

Reply:
Ok, great. Do we want to consider removing the host and the cloud provider metadata that gets added with `sync_host_metadata` and resource detection enabled? Some users might want to keep the istio labels but be okay with these being removed esp, given this is info they can easily look up with the pod/node name.

Reply:
Host and the cloud provider metadata is essential for the correlation with all other metrics and cloud integrations, I would keep them enabled by default across all the metrics that are being sent by the collector instead of removing them conditionally for istio metrics. Removing these 6 attribute should give us enough room